### DISCUSSION TITLE
Payer pour changer le nom d'une guilde

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#580] [Nouvelles idées]
Pouvoir payer 5000 $ pour changer le nom d'une guilde

**Comment by romain22222** [comment]
notes si acccptée : penser à changer le nom de guilde dans les logs

**Reply by BastLast** [reply]
tu estime à quoi le coût en dev ? c'est juste un update en bdd d'une ligne ? ou faut rechercher à pleins d'endroits ?

**Reply by romain22222** [reply]
deux updates, une sur chaque bdd : la bdd de logs a été pensée pour être indépendante et compact

**Comment by Voisin1234** [comment]
#2387

**Comment by Voisin1234** [comment]
Du coup, au fil de toutes les suggestions qui ont été pour le changement de guilde. Je pense que cela est intéressant et recherché par les joueurs. J’ai mis plus haut le lien de mon idée également, qui, dans celle-ci n’a pas les mêmes prix pour changer de nom. Sur ce bonne soirée.

**Reply by BastLast** [reply]
yep pas de soucis après dans ce genre d'idée le prix c'est jamais important c'est toujours revus à l'implémentaiton en fonction de l'équilibrage voulu

**Reply by BastLast** [reply]
pour l'instant on est full focus sur un autre projet alors on peut pas faire ce genre de suggestions mais dès qu'on pourra, on fera ce genre d'idée, si c'est important pour vous go upvote et partager, dès qu'on traitera les suggestions, les upvotes joueront beaucoup donc go voter pour toutes les idées qui vous plaisent
